### DISCUSSION TITLE
What installer type does BISQ use?

### DISCUSSION THREAD

**Discussion by anderlli0053** [bisq-network/bisq#6982] [Q&A] [answered]
Hello!

I am creating Scoop manifest for Bisq and i don't know what installer type it uses.

I tried everything that i could detect, even WIX, Burn, .... used INNOUNP, DARK, LESSMSI, .... but nothing seems to work as it ain't common known installer type. The only thing i could find is from WinGet app manifest that lists **_InstallerType: burn_** but that's no help at all as i already mentioned burn extractor doesn't work!

**Comment by alvasw** [comment, marked as answer]
Hi @anderlli0053,
Bisq uses the WiX v3 Toolset (https://wixtoolset.org/docs/wix3/).

**Reply by anderlli0053** [reply]
> Hi @anderlli0053, Bisq uses the WiX v3 Toolset (https://wixtoolset.org/docs/wix3/).

Thanks!
